MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -39,15 +42,56 @@ class QuestionFactory: QuestionFactoryProtocol {
                                                           correctAnswer: false),
                                              QuizQuestion(image: "Vivarium",
                                                           text: "Рейтинг этого фильма больше чем 6?",
-                                                          correctAnswer: false)]
+                                                          correctAnswer: false)] */
 
     func requestNextQuestion() {
-        let index = (0..<questions.count).randomElement() ?? 0
-        let question = questions[safe: index]
-        delegate.didReceiveNextQuestion(question: question)
+        DispatchQueue.global().async { [weak self] in
+            guard let self = self else { return }
+            let index = (0..<self.movies.count).randomElement() ?? 0
+
+            guard let movie = self.movies[safe: index] else { return }
+
+            var imageData = Data()
+
+           do {
```

Comment:
Не забывайте использовать форматтер `control+i`
```suggestion
            do {
```